packages/button/components/ButtonBase.tsx

```diff
@@ -98,41 +102,66 @@ class ButtonBase extends React.PureComponent<ButtonBaseProps, {}> {
       isProcessing,
       isFullWidth,
       onClick,
+      url,
       ...other
     } = this.props;
 
-    return (
-      <FocusStyleManager
-        focusEnabledClass={focusStyleByAppearance(appearance, isInverse)}
-      >
+    const buttonClassName = cx(
+      buttonReset,
+      button(appearance),
+      buttonBase,
+      textWeight("medium"),
+      className,
+      {
+        [fullWidthButton]: isFullWidth,
+        [buttonInverse(appearance)]: isInverse,
+        [getMutedButtonStyles(appearance)]: disabled || isProcessing,
+        [getInverseMutedButtonStyles(appearance)]:
+          (disabled || isProcessing) && isInverse
+      }
+    );
+
+    const getButtonNode = () => {
+      if (url) {
+        return !disabled && !isProcessing ? (
+          <a
+            href={url}
+            className={buttonClassName}
+            onClick={this.onClick}
+            tabIndex={0}
+          >
+            {this.getButtonContent()}
+          </a>
+        ) : (
+          // tslint:disable react-a11y-anchors
+          //
+          // this rule was erroring because the anchor had no href, but in this
+          // case that is intentional beacuse it's disabled
+          <a className={buttonClassName} aria-disabled="true" tabIndex={-1}>
```

Comment:
how about using a `button`-tag instead for the case no url is given and it should be disabled?